libreantdb/api.py

```diff
@@ -359,17 +361,15 @@ def autocomplete(self, fieldname, start):
     # End queries }}}
 
     # Operations {{{2
-    def add_book(self, **book):
+    def add_book(self, body, doc_type='book'):
```

Comment:
Doesn't this change require corresponding change in `archivant.insert_volume`? I mean, I know that the test passes, but I thought that changing a method prototype would have  required changes in callers.

Reply:
I just specialized the function parameter according to the invocation inside `insert_volume`. Thus it is all coherent.